components/admin_console/manage_roles_modal/manage_roles_modal.tsx

```diff
@@ -240,9 +243,13 @@ export default class ManageRolesModal extends React.PureComponent<Props, State>
                                     disabled={this.state.isSystemAdmin}
                                     onChange={this.handleUserAccessTokenChange}
                                 />
-                                <FormattedMarkdownMessage
+                                <FormattedMessage
                                     id='admin.manage_roles.allowUserAccessTokens'
-                                    defaultMessage='Allow this account to generate [personal access tokens](!https://developers.mattermost.com/integrate/admin-guide/admin-personal-access-token/).'
+                                    defaultMessage='Allow this account to generate <link>personal access tokens</link>.'
+                                    values={{
+                                        link: (msg: React.ReactNode) => (<a href='https://developers.mattermost.com/integrate/admin-guide/admin-personal-access-token' referrer='noreferrer' target='_blank'>{msg}</a>),
```

Comment:
nit: now that we are using the URLS this way, shouldn't we place all this URL values in constants? @hmhealey thoughts?

Reply:
Yeah I noticed quite some other urls in other strings. Maybe it is indeed a good idea to modify to this format.
I'm willing to do so an a few next prs.